set.mm

```diff
@@ -77236,17 +77235,16 @@ currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
     $( Necessary and sufficient condition for ` dom tpos F ` to be a relation.
        (Contributed by Mario Carneiro, 10-Sep-2015.) $)
     reldmtpos $p |- ( Rel dom tpos F <-> -. (/) e. dom F ) $=
-      ( vy vx cdm wrel c0 wcel wn cv wbr wex 0ex wb cvv elv wss wi df-rel sylbi
-      eldm ctpos brtpos0 cxp 0nelxp ssel mtoi breldm nsyl3 sylbir exlimiv con2i
-      vex wa ccnv csn relcnv sseli a1i elsni breq1d pm2.24d syl6bi com3l impcom
-      mpbi wo cun cuni brtpos2 simplbi elun sylib adantl mpjaod ex syl5bi ssrdv
-      exlimdv sylibr impbii ) AUAZDZEZFADZGZHZWEWCWEFBIZAJZBKWCHZBFALTWHWIBWHFW
-      GWAJZWIWJWHMBWGANUBOZWCFWBGZWJWCWLFNNUCZGZNNUDWCWBWMPZWLWNQWBRZWBWMFUESUF
-      FWGWALBULZUGUHUIUJSUKWFWOWCWFCWBWMCIZWBGWRWGWAJZBKWFWRWMGZBWRWACULTWFWSWT
-      BWFWSWTWFWSUMZWRWDUNZGZWTWRFUOZGZXCWTQXAXBWMWRXBEXBWMPWDUPXBRVEUQURWSWFXE
-      WTQXEWSWFWTXEWSWJWFWTQZXEWRFWGWAWRFUSUTWJWHXFWKWHWEWTFWGALWQUGVASVBVCVDWS
-      XCXEVFZWFWSWRXBXDVGGZXGWSXHWRUOUNVHWGAJZWSXHXIUMMBWRWGANVIOVJWRXBXDVKVLVM
-      VNVOVRVPVQWPVSVT $.
+      ( vy vx cdm wrel c0 wcel wn cv wbr wex 0ex wb cvv vex breldm sylbi wss wi
+      eldm ctpos brtpos0 elv 0nelrel0 nsyl3 sylbir exlimiv con2i wa ccnv relcnv
+      cxp csn df-rel mpbi sseli a1i elsni breq1d pm2.24d syl6bi com3l impcom wo
+      cun cuni brtpos2 ax-mp simplbi elun sylib adantl mpjaod ex exlimdv syl5bi
+      ssrdv sylibr impbii ) AUAZDZEZFADZGZHZWDWBWDFBIZAJZBKWBHZBFALTWGWHBWGFWFV
+      TJZWHWIWGMBWFANUBUCZWBFWAGWIWAUDFWFVTLBOZPUEUFUGQUHWEWANNULZRWBWECWAWLCIZ
+      WAGWMWFVTJZBKWEWMWLGZBWMVTCOTWEWNWOBWEWNWOWEWNUIZWMWCUJZGZWOWMFUMZGZWRWOS
+      WPWQWLWMWQEWQWLRWCUKWQUNUOUPUQWNWEWTWOSWTWNWEWOWTWNWIWEWOSZWTWMFWFVTWMFUR
+      USWIWGXAWJWGWDWOFWFALWKPUTQVAVBVCWNWRWTVDZWEWNWMWQWSVEGZXBWNXCWMUMUJVFWFA
+      JZWFNGWNXCXDUIMWKWMWFANVGVHVIWMWQWSVJVKVLVMVNVOVPVQWAUNVRVS $.
```

Comment:
The proof size shrinks from 44 down to 41 lines.  No change in Syntax, Axioms and Definitions usage.